packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-config/spark-config.scss

```diff
@@ -0,0 +1,17 @@
+.sparkconfig__arguments {
+  padding: 0.5rem 0;
+  border: 0;
+  width: 30%;
+}
+
+.oui-field__warning-show {
```

Comment:
Why do you want to change the colors ? The theme is there so you don't have to think about colors and stuff 😄